scripts/iota_cache.py

```diff
@@ -0,0 +1,67 @@
+import time
```

Comment:
Add "import datetime" for TagHelper class.

Reply:
This logic should be separated from this class